src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalDayPrice = 40;
+  const mediumDiscount = 20;
+  const bigDiscount = 50;
```

Comment:
Those values are actual constants (Like `π` in math), we are defining their names in UPPERCASE.
<img width="147" alt="image" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/49764881/334fb018-f1ac-48d3-a200-ab1e628c7596">